### DISCUSSION TITLE
Connect Immich to existing Immich library

### DISCUSSION THREAD

**Discussion by rafsko1** [immich-app/immich#6041] [General]
Hi all, hope you can guide me in the right direction. I have RPi 4 with Debian on sd card that crashed and Immich library on the mounted hdd. Now created usb stick with Debian. Will i just provide the path to the library on the hdd and scan?

**Comment by bo0tzz** [comment]
If you lost the database volume and you don't have backups, then you'll need to start from scratch by uploading everything again. Alternatively you can import the files as an external library.

**Reply by rafsko1** [reply]
Have database volume on hdd. Just debian crashed on sd card.
